profiler/src/Demos/Samples.Computer01/Obfuscation.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.IO;
+using System.Threading;
+using Aspose.Pdf;
+
+namespace Samples.Computer01
+{
+    internal class Obfuscation : ScenarioBase
+    {
+        public override void OnProcess()
+        {
+            try
+            {
+                Document pdf_doc = new Document();
+                Page page = pdf_doc.Pages.Add();
+                page.Paragraphs.Add(new Aspose.Pdf.Text.TextFragment("Line1"));
+                page.Paragraphs.Add(new Aspose.Pdf.Text.TextFragment("Line2"));
+                page.Paragraphs.Add(new Aspose.Pdf.Text.TextFragment("Line3"));
+                page.Paragraphs.Add(new Aspose.Pdf.Text.TextFragment("Line4"));
+                page.Paragraphs.Add(new Aspose.Pdf.Text.TextFragment("Line5"));
+                page.Paragraphs.Add(new Aspose.Pdf.Text.TextFragment("Line6"));
+
+                var filename = Path.GetTempFileName() + ".pdf";
+                Console.WriteLine(filename);
+                pdf_doc.Save(filename);
+                File.Delete(filename);
+            }
+            catch (Exception x)
+            {
+                Console.WriteLine(x.Message);
+            }
+        }
+
+        public override void Run()
```

Comment:
Shouldn't you add a case to `Obfuscation.Run`  to the `switch` statement in the `Run` function in `ComputerService.cs` ?

Reply:
Good catch!
✅ done